prac_08/dynamic_labels.py

```diff
@@ -0,0 +1,27 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.uix.label import Label
+
+
+class DynamicLabelsApp(App):
+    """App for displaying dynamic labels for a list of names."""
+
+    def __init__(self, **name):
+        super().__init__(**name)
+        self.names = ['dog', 'Cat ', 'bird']
+
+    def build(self):
+        """Build the Kivy GUI."""
+        self.title = "Dynamic Labels"
+        self.root = Builder.load_file('dynamic_labels.kv')
+        self.create_names()
```

Comment:
"The widgets (Buttons in the demo, but yours will be Labels) are made with a loop in the build function"
This is one of the dot points from the Dynamic Labels activity, instead of making the names in a separate function we just needed to have it in the Build method